lib/resty/etcd/v3.lua

```diff
@@ -44,11 +46,25 @@ local unmodifiable_headers = {
 local refresh_jwt_token
 
 
-local function choose_endpoint(self)
+local function ring_balancer(self)
     local endpoints = self.endpoints
+    local endpoints_len = #endpoints
+
+    self.init_count = (self.init_count or -1) + 1
```

Comment:
Would it be better if the init_count starts from a random number?

Reply:
update, now we can specify `init_count` when creating etcd cli, otherwise it is chosen randomly from [0, 100].

Reply:
@tzssangglass 
We can remove `or -1` now as init_count is always set before this?

Reply:
rm it